### PR TITLE
Fix excel imports

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/contacts/models.py [hunk 1]
```diff
@@ -1129,6 +1129,11 @@ def import_excel(cls, filename, user, import_params, log=None, import_results=No
             # trim all our values
             row = [cls.normalize_value(cell_value) for cell_value in row]
 
+            row = [str(cell_value)
+                   if not isinstance(cell_value, datetime.date) and not isinstance(cell_value, datetime.datetime)
+                   else cell_value
+                   for cell_value in row]
+
```
Comment (rowanseymour) [inline temba/contacts/models.py]: also, if you can't fit a list comprehension onto 1 line, then you should probably use a regular for loop

File: temba/flows/tests.py [hunk 2]
```diff
@@ -544,6 +544,10 @@ def test_export_results(self):
 
         self.create_group('Devs', [self.contact])
 
+        # contact name with an illegal character
+        self.contact3.name = "Nor\02bert"
```
Comment (rowanseymour) [inline temba/flows/tests.py]: Maybe have this test check that name can have non-ascii chars, and then check output in Excel file

Comment (norkans7) [inline temba/flows/tests.py]: That's why I modified this name the excel file is checked as before and should "Norbert"

File: temba/contacts/models.py (hunk 1 above)
Comment (norkans7) [inline temba/contacts/models.py]: this is to convert numbers into strings if a user import with name cell value like 3 the import was throwing so we need to make all the data as string except the dates and times

File: temba/flows/tests.py (hunk 2 above)
Comment (rowanseymour) [inline temba/flows/tests.py]: Ok cool - just want to check that `remove_control_characters` handles non-ascii characters ok. Not sure if we test that anywhere. If not, you can be "Norbért"

**Pull request merged**